include/picongpu/simulation_defines/param/laser.param

```diff
@@ -369,6 +369,108 @@ namespace picongpu
         constexpr float_X LASER_PHASE = 0.0;
     }
 
+    namespace laserExpRampWithPrepulse
+    {
+    /** Laser profile with gaussian spatial envelope and the following temporal
```

Comment:
pls indent correctly the code inside this namespace by 4 further spaces (see existing code above)

Reply:
please fix the comment style, it is invalid C++ syntax.
C++ multi-line comments are of form:
```C++
/* lalala
 * lala
 */
```

Also, do **not** use doxygen opening `/**` here since this block is not bound to an object/a single variable properly.